src/SFA.DAS.Notifications.Infrastructure/NotifyEmailService/NotifyClientWrapper.cs

```diff
@@ -58,7 +58,6 @@ private async Task SendMessage(NotifyMessage content, CommunicationType communic
             catch (NotifyClientException notifyClientException)
             {
                 Logger.Error(notifyClientException, $"Error sending communication {communicationType.ToString()} to Gov Notify with Gov.Notify Client");
-                throw;
```

Comment:
This will suppress the error. The message will be completed (seemingly successful) and there will be no retries. Which errors are we suppressing here?

Reply:
We don't know the reason for failure but the error is logged and NotifyClientException are filling the NServiceBus error queue. NotifyClientExceptions are used in the execution policy for retries from what I can see.

Reply:
Could we change it to only suppress on particular error messages? Not the most elegant solution I know. But would allow us to only suppress those that aren't worth adding to the error queue. Phone number incorrect format